app/services/feeds/import.rb

```diff
@@ -23,27 +23,30 @@ def call
 
       users.in_batches(of: users_batch_size) do |batch_of_users|
         feeds_per_user_id = fetch_feeds(batch_of_users)
-        Rails.logger.error("feeds::import::feeds_per_user_id.length: #{feeds_per_user_id.length}")
+        DatadogStatsClient.count("feeds::import::fetch_feeds.count", feeds_per_user_id.length)
 
         feedjira_objects = parse_feeds(feeds_per_user_id)
-        Rails.logger.error("feeds::import::feedjira_objects.length: #{feedjira_objects.length}")
+        DatadogStatsClient.count("feeds::import::parse_feeds.count", feedjira_objects.length)
 
         # NOTE: doing this sequentially to avoid locking problems with the DB
         # and unnecessary conflicts
         articles = feedjira_objects.flat_map do |user_id, feed|
           # TODO: replace `feed` with `feed.url` as `RssReader::Assembler`
           # only actually needs feed.url
           user = batch_of_users.detect { |u| u.id == user_id }
-          create_articles_from_user_feed(user, feed)
+
+          DatadogStatsClient.time("feeds::import::create_articles_from_user_feed", tags: ["user_id:#{user_id}"]) do
```

Comment:
I have never used the `time` stat here, interested to see how it works! If we find the Datadog metrics not giving us enough insight or cumbersome to work with I would suggest changing this to a Honeycomb span.